compiler-rt/lib/builtins/cpu_model/aarch64/fmv/mrs.inc

```diff
@@ -52,23 +45,20 @@ static void __init_cpu_features_constructor(unsigned long hwcap,
     setCPUFeature(FEAT_FCMA);
   if (hwcap & HWCAP_SB)
     setCPUFeature(FEAT_SB);
-  if (hwcap & HWCAP_SSBS)
+  if (hwcap & HWCAP_SSBS) {
+    setCPUFeature(FEAT_SSBS);
```

Comment:
Does SSBS2 imply SSBS? If so, I think I need to add `CHECK_BIT(CAP_BIT_FEAT_SSBS, FEAT_SSBS);` to `apple.inc`

Reply:
I'll go dig through our docs to double check.

Reply:
From the Arm Architecture Reference Manual on the documentation of `ID_AA64PFR1_EL1.SSBS`:
"FEAT_SSBS2 implements the functionality identified by the value 0b0010".

As far as I understand HWCAP_SSBS is set based on that value. Here is one resource I found https://www.kernel.org/doc/html/next/arm64/elf_hwcaps.html
> HWCAP_SSBS
> Functionality implied by ID_AA64PFR1_EL1.SSBS == 0b0010.

Reply:
Never mind. They are independent features as far as FMV should be concerned, and M1 only implements SSBS2.

Reply:
Oh wow, that wording is confusing (the Arm ARM's, not yours).